CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## [Version 1.2.1] - 2022-09-12
+
+- Corrected the default for `no_data_value` in `ImportFromTiffTask` and `ExportToTiffTask` to `None`. The previous default of `0` was a poor choice in many scenarios. The switch might alter behavior in existing code.
+- Added `ExplodeBandsTask` that explodes a multi-band feature into multiple features.
+- Exposed resampling parameters in Sentinel Hub tasks and included a `geometry` execution parameter.
+- Reworked internal classes `FeatureIO` and `_FeatureDict` to improve types and maintainability.
+- Fixed y-axis orientation of `MeteoblueRasterTask`.
+- `FilterTimeSeriesTask` adjusted to work with multiprocessing.
+- EOPatch plotting no longer anti-aliases by default (removes issues with phantom values in mask plots)
+- Improved documentation building, fixing a few broken links.
+
```

Comment:
Also missing? 

* The default behaviour of `EOPatch.plot()` is now to plot without interpolation.
* `SpatialResizeTask` can now also accept the target resolution as the parameter. Refactored the way the parameters are passed to this function.

Reply:
The first point is supposed to be line 9. Will add the second one.

Reply:
:facepalm: missed the anti-aliasing. But once you add the second one I think it's good.